### DISCUSSION TITLE
Questions about scalefactor

### DISCUSSION THREAD

**Discussion by carlogico** [karttapullautin/karttapullautin#153] [Q&A]
I assumed that the `scalefactor` parameter only would influence the x and y step of the height grid creation (obviously that would need to be taken into account when computing steepness), but it seems to affect the contour interval passed to `heightmap2contours`, which seems quite odd to me. Is this intended? And if so, what is the purpose of that?

**Comment by Jagge** [comment]
that was implemented to make more generalized maps, for example scale 1:30 000 with 10 m interval + contours. Without having to tweak point could input.
